src/views/requestAccess.js

```diff
@@ -88,25 +78,24 @@ export const RoleDropDown = () => {
 	return (
 		<div className={bottomPad} >
 			<div className={dropDown}>
-			  <div className="dropdown-trigger">
-			    <button className="button is-rounded drop-button" aria-haspopup="true" aria-controls="dropdown-menu" onClick={handleClickDropdown} >
-			      <span className="drop-button-content"> {dropContent} </span>
-			      <span className={dropButton} >
-			        <i className="fas fa-angle-down" aria-hidden="true"></i>
-			      </span>
-			    </button>
-			  </div>
-			  <div className="dropdown-menu dropdown-menu-place">
-			    <div className="dropdown-content dropdown-content-width">
-			      <DropDownItem dropContent="JOIN Staff" clickHandle={handleClickJOIN} />
-			      <DropDownItem dropContent="Property Manager" clickHandle={handleClickProperty} />
+			    <div className="dropdown-trigger">
+				    <button className="button is-rounded drop-button" aria-haspopup="true" aria-controls="dropdown-menu" onClick={handleClickDropdown} >
+					    <span className="drop-button-content"> {dropContent} </span>
+					    <span className={dropButton} >
+					    	<i className="fas fa-angle-down" aria-hidden="true"></i>
+					    </span>
+				    </button>
+			    </div>
+			    <div className="dropdown-menu dropdown-menu-place">
+				  	<div className="dropdown-content dropdown-content-width">
+					    <DropDownItem dropContent="JOIN Staff" clickHandle={handleClickJOIN} />
+					    <DropDownItem dropContent="Property Manager" clickHandle={handleClickProperty} />
+				  	</div>
 			    </div>
-			  </div>
 			</div>
-
 			<div>
-				{propertyPage && <PropertyPage />}
-				{joinPage && <JoinPage />}
+				{propertyPage && <PropertyManagerOptions />}
```

Comment:
We can also change the names of these booleans, like `viewPropertyManagerOptions` and `viewJoinStaffOptions`